jobrunner/service.py

```diff
@@ -0,0 +1,55 @@
+"""
+Script runs both jobrunner flows in a single process.
+"""
+import datetime
+import logging
+import time
+import threading
+
+from .log_utils import configure_logging, set_log_context
+from . import config
+
+
+from . import run
+from . import sync
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    """Run the main run loop after starting the sync loop in a thread."""
+
+    # load any env file
+    path = Path(os.environ.get('ENVPATH', '.env'))
+    if path.exists():
+        log.info(f'Loading environment variables from {path}')
+        env = parse_env(path.read_text())
+        if env:
+            os.environ.update(env)
+
+    try:
+        log.info("jobrunner.service started")
+        # daemon=True means this thread will be automatically join()ed when the
```

Comment:
Ah yes, I always forget that this flag has the exact opposite meaning from the one I expect it to have